Apps/Contoso.WinUI.Desktop.Demo/Contoso.WinUI.Desktop.Demo/Properties/PublishProfiles/win10-x64.pubxml

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+https://go.microsoft.com/fwlink/?LinkID=208121.
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PublishProtocol>FileSystem</PublishProtocol>
+    <Platform>x64</Platform>
+    <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
+    <PublishDir>bin\$(Configuration)\$(TargetFramework)\$(RuntimeIdentifier)\publish\</PublishDir>
+    <SelfContained>true</SelfContained>
+    <PublishSingleFile>False</PublishSingleFile>
+    <PublishReadyToRun>True</PublishReadyToRun>
+   <!-- 
+    See https://github.com/microsoft/CsWinRT/issues/373
```

Comment:
Probably add some notes in which cases we would want to uncomment this line.

Reply:
jfyi
this is just a default comment that is added after the application is created